client/src/app/containers/Home/ActiveGames.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import { IconButton, TableCell, TableRow } from '@material-ui/core';
+import PersonPinCircleIcon from '@material-ui/icons/PersonPinCircle';
+import SportsEsportsIcon from '@material-ui/icons/SportsEsports';
+
+import { User } from '../../../auth/slice';
+import { colorToHue } from '../../../utils/colorToHue';
+import { ActiveGame } from './slice';
+
+const getHueForPlayerInGame = (player: User, game: ActiveGame) => {
+    const color = game.players.find(p => p.id === player.id)?.color;
+    return colorToHue(color ?? '');
+};
+
+interface Props {
+    player: User;
+    games: ActiveGame[];
+    onClickPlay: (game: ActiveGame) => void;
+}
+
+const ActiveGames: React.FunctionComponent<Props> = ({
+    player,
+    games,
+    onClickPlay,
+}) => {
+    return (
+        <>
```

Comment:
:mag: what is happening here?

Reply:
`<>` is equivalent to `<Fragment>`. `Fragment` exists because a react component must return exactly _one_ element (which can have multiple children that are siblings), so can be used in the following case:
```jsx
// Bad time!
return (
    <Button />
    <Button />
);

// Better time!
return (
    <>
        <Button />
        <Button />
    </>
);
```

But, it must be leftover because we don't need it here.

Reply:
I lied. We do need it here, because otherwise the `map` is returning multiple sibling elements without a parent (like in the first case above)